service/lib/dinstaller/dbus/storage/manager.rb

```diff
@@ -73,6 +76,10 @@ def finish
 
         # @return [DInstaller::Software::Manager]
         attr_reader :backend
+
+        def register_proposal_callbacks
+          backend.proposal.add_on_change_listener { update_validation }
```

Comment:
NP: This line should be adapted https://github.com/yast/d-installer/blob/master/service/lib/dinstaller/storage/proposal.rb#L62

Reply:
```suggestion
          backend.proposal.on_calculate { update_validation }
```